apps/metadata-utils/src/types.ts

```diff
@@ -85,3 +88,10 @@ export interface ISchemaMetaData {
   description?: string;
   tables: ITableMetaData[];
 }
+
+export interface IFieldError {
+  message: string;
+}
+
+export type columnId = string;
+export type columnValue = string | number | boolean | Object;
```

Comment:
Record<string,any> instead of Object is a bit more consistent with the rest of the code.